testutil/validatormock/propose.go

```diff
@@ -165,8 +171,14 @@ func ProposeBlindedBlock(ctx context.Context, eth2Cl eth2wrap.Client, signFunc S
 		return err
 	}
 
-	var pubkey eth2p0.BLSPubKey
-	var block *eth2api.VersionedBlindedBeaconBlock
+	if len(duties) == 0 {
```

Comment:
This method and `ProposeBlock` are similar in what they do (fetch proposer duties, find/create the associated block proposal) yet they have different code flows.

IMO this check for duties presence should be ported to `ProposeBlock` as well, to short-circuit the method.

Reply:
Done!